README.md

```diff
@@ -1,2 +1,28 @@
 # github-actions
+
 Public repository for reusable github actions 
+
+As is customary for GitHub Actions, we provide release tags for you to reference in your repository's workflow files. The `v1` tag is a moving tag that will always apply to the latest version 1 train release.
```

Comment:
We will need to use the semantic versioning I imagine as the tags are immutable

Reply:
Git tags are mutable, it's just generally not nice. I was going for `v1.0.0` and `v1` (where `v1` could be re-tagged).

Reply:
ah ok I assumed they couldn't be recreated to point to a different commit apparently they can, there are tag protection rules you can enable tho interesting